File: Mathematics/__1000__/1161/1161.php
<?php 

    function fatorial ($n){
        if($n == 1 || $n == 0){
            return 1;
        }
        return $n * fatorial($n - 1);
    }

    while(($line = readline()) != false){

        if(trim($line) == '') break;

        list($a, $b) = explode(' ', $line);

        $a = (int) $a;
        $b = (int) $b;

        $fa = fatorial($a);
        $fb = fatorial($b);

        $soma = $fa + $fb;

        echo "$soma\n";
    }
?>

File: Mathematics/__1000__/1161/1161.cpp
#include <iostream>

using namespace std;

long fatorial(int n){
    if(n == 1 || n == 0){
        return 1;
    }
    return n * fatorial(n - 1);
}

int main() {

    int a, b;
    while(cin >> a >> b){
        long fa = fatorial(a);
        long fb = fatorial(b);

        long soma = fa + fb;
        cout << soma << endl;
    }

    return 0;
}